src/Symfony/Component/Console/Output/Output.php

```diff
@@ -140,7 +140,8 @@ public function writeln($messages, $options = self::OUTPUT_NORMAL)
     public function write($messages, $newline = false, $options = self::OUTPUT_NORMAL)
     {
         $messages = (array) $messages;
-
+        $options = is_numeric($options) ? (int) $options : 0;
```

Comment:
If we expected an `integer`, but got something different, then it is a contract violation and `InvalidArgumentException` shall be raise instead of silently ignoring the violation.